edx_proctoring/views.py

```diff
@@ -328,35 +323,6 @@ def get(self, request, attempt_id):
                 )
                 raise ProctoredExamPermissionDenied(err_msg)
 
-            # check if the last_poll_timestamp is not None
-            # and if it is older than SOFTWARE_SECURE_CLIENT_TIMEOUT
-            # then attempt status should be marked as error.
-            last_poll_timestamp = attempt['last_poll_timestamp']
-
-            # if we never heard from the client, then we assume it is shut down
-            attempt['client_has_shutdown'] = last_poll_timestamp is None
-
-            if last_poll_timestamp is not None:
-                # Let's pass along information if we think the SoftwareSecure has completed
-                # a healthy shutdown which is when our attempt is in a 'submitted' status
-                if attempt['status'] == ProctoredExamStudentAttemptStatus.submitted:
-                    attempt['client_has_shutdown'] = has_client_app_shutdown(attempt)
-                else:
-                    # otherwise, let's see if the shutdown happened in error
-                    # e.g. a crash
-                    time_passed_since_last_poll = (datetime.now(pytz.UTC) - last_poll_timestamp).total_seconds()
-                    if time_passed_since_last_poll > constants.SOFTWARE_SECURE_CLIENT_TIMEOUT:
-                        try:
-                            update_attempt_status(
-                                attempt['proctored_exam']['id'],
-                                attempt['user']['id'],
-                                ProctoredExamStudentAttemptStatus.error
-                            )
-                            attempt['status'] = ProctoredExamStudentAttemptStatus.error
-                        except ProctoredExamIllegalStatusTransition:
-                            # don't transition a completed state to an error state
-                            pass
-
             # add in the computed time remaining as a helper to a client app
```

Comment:
The term "client app" is confusing in the case of proctoring.  I'd just delete this comment altogether.  Feel free to do that with the squash.